### PR TITLE
Log in

### PR DESCRIPTION
**Pull request by kalex19**

### CODE REVIEW

File: Components/CreateAccount/CreateAccount.js [hunk 1]
```diff
@@ -102,7 +108,11 @@ export class CreateAccount extends Component {
 		try {
 			const response = await fetch('https://sophia-be.herokuapp.com/api/v1/clients/', options);
 			const account = await response.json();
-      await this.logInClient(account.username, account.password)
+			// await logInClient(account.username, account.password)
+			// console.log(user)
+			// this.props.logIn(user)
+				this.setState(initialState);
+    	this.props.navigation.navigate('ClientProfile');
```
Comment (andreeahanson) [inline Components/CreateAccount/CreateAccount.js]: @kalex19 , 

This 'navigate' takes the user directly to ClientProfile right after he/she/they log in? Shouldn't it take them to the 'Homepage', where they get to pick whether they want to look at their profile or their lists?

File: Components/IndividualList/IndividualList.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { View, Text, StyleSheet, TextInput } from "react-native";
 import { TouchableHighlight } from "react-native-gesture-handler";
 import { connect } from "react-redux";
 import { loadTasks } from "../../actions";
-import { fetchTasks, postTask, patchTask, deleteTask } from "../../apiCalls";
+import { fetchTasks, postTask, patchTask, deleteTask } from "../../Utils/apiCalls";
```
Comment (andreeahanson) [inline Components/IndividualList/IndividualList.js]: @kalex19 ,

So far I've added all my fetches in a file called apiCalls.js . I see that you've added yours in a Utils folder. Should we put them all in the same place or, should we leave them separate, as they are?

**Comment by andreeahanson** [comment]
@kalex19,

really nice job on the login and createAccount! Thank you for taking care of those.

File: Components/CreateAccount/CreateAccount.js (hunk 1 above)
Comment (kalex19) [reply Components/CreateAccount/CreateAccount.js]: Yep, it goes to User now.

File: Components/IndividualList/IndividualList.js (hunk 2 above)
Comment (kalex19) [reply Components/IndividualList/IndividualList.js]: I think ideally they should be in the same place. We can refactor later.

**Pull request merged**